tests/functional/aws-node-sdk/test/bucket/testBucketStress.js

```diff
@@ -0,0 +1,55 @@
+import { S3 } from 'aws-sdk';
+import { times, timesSeries, waterfall } from 'async';
+
+import getConfig from '../support/config';
+
+const bucket = `stress-test-bucket-${Date.now()}`;
+const text = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890';
+const objectCount = 100;
+const loopCount = 10;
+
+function putObjects(s3, cb) {
+    let count = 0;
+    times(objectCount, (i, next) => {
+        const params = { Bucket: bucket, Key: `foo${i}`, Body: text };
+        s3.putObject(params, error => {
+            if (error || ++count === objectCount) {
+                cb(error);
+            }
+        });
+        next();
+    }, () => {});
+}
+
+function deleteObjects(s3, cb) {
+    let count = 0;
+    times(objectCount, (i, next) => {
+        const params = { Bucket: bucket, Key: `foo${i}` };
+        s3.deleteObject(params, error => {
+            if (error || ++count === objectCount) {
+                cb(error);
+            }
+        });
+        next();
+    }, () => {});
+}
+
+describe('aws-node-sdk stress test bucket', function testSuite() {
+    this.timeout(120000);
+    let s3;
+    // setup test
+    before(done => {
+        const config = getConfig('default', { signatureVersion: 'v4' });
+        s3 = new S3(config);
+        setTimeout(done, 1000); // wait for bucketclient to query metadata
```

Comment:
I'm unclear why we need this.  If we need this here, don't we need this in all tests?  If we need a delay before running tests because of the new metadata wrapper init, we should put that delay in the circle.yml and in integration.